src/earn/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import { Network, NetworkId } from 'src/transactions/types'
+import { INTERNAL_RPC_SUPPORTED_NETWORKS } from 'src/viem'
+import { networkIdToNetwork } from 'src/web3/networkConfig'
+
+/**
+ * Helper function to check if gas is subsidized for the given networkId
+ * @param networkId - networkId to check
+ * @returns boolean
+ */
+export function isGasSubsidizedForNetwork(networkId: NetworkId) {
+  return (
+    (INTERNAL_RPC_SUPPORTED_NETWORKS as readonly Network[]).includes(
+      networkIdToNetwork[networkId]
+    ) && getFeatureGate(StatsigFeatureGates.SUBSIDIZE_STABLECOIN_EARN_GAS_FEES)
```

Comment:
We can eventually do this via dynamic config, but this is good enough for now

Reply:
Is there a plan for which ticket we will take care of it? just don't want to forget

Reply:
no ticket as of now. Until we plan to support this for other networks, which will need more wallet changes, this can wait.